test/test_project.py

```diff
@@ -72,10 +82,10 @@ def test_update_copy_fields(self):
             m.put(self.baseurl + '/1d0304cd-3796-429f-b815-7258370b9b74', text=response_xml)
             single_project = TSC.ProjectItem('test')
             single_project._id = '1d0304cd-3796-429f-b815-7258370b9b74'
-            single_project._permissions = 'Test to check if permissions copied over.'
+            single_project._some_field = 'Test to check if fields are copied over.'
```

Comment:
Why this change?

`_some_field` isn't an attribute of the model, actually, neither is `_permissions`. 

I think you can just delete this from the test entirely